pkg/limits/rate_limiting.go

```diff
@@ -153,6 +155,12 @@ var configs = []counterConfig{
 		Limit:  30,
 		Period: 1 * time.Hour,
 	},
+	// JobNotesPersistType
+	{
+		Prefix: "job-notes-persist",
+		Limit:  100,
+		Period: 1 * time.Hour,
```

Comment:
Does that mean we'll persist notes on disk only once every hour? If we want to sync those files with Desktop we should expect people to think the version they have on disk is the latest one after they're done editing their note.

Reply:
No, we are in the rate-limiting package. It means that an instance can't create more than 100 jobs of the type `job-notes-persist` in an hour.

Reply:
Ah right I missed that. Thanks.